README.md

```diff
@@ -40,6 +40,27 @@ More details can be found [here](https://forum.cow.fi/t/grant-application-decent
 
 - During development use `make formatters`, `make code-checks` and `make generators`
 
+
 - After building your development environment and pulling the packages, you can conduct an end-to-end test with a local network and 4 agents by running
       
       pytest packages/valory/agents/decentralized_watchtower/tests/test_agents/test_decentralized_watchtower.py::TestWatchtowerFourAgents
+
+## Healthcheck for the agent
+
+The agents offer an http endpoint. It listens on port 8000. Once you have an agent running, you can check it's health status by calling the /healthcheck endpoint. 
+Example:  
+```bash
+curl http://localhost:8000/healthcheck
```

Comment:
localhost or agent address?

Reply:
Well assuming its running on localhost, then that's how it is. Obviously if its running elsewhere, that should be reflected.

Reply:
that should be clear, no?